server/Route.js

```diff
@@ -1,5 +1,55 @@
 const express = require('express');
+const db = require('./db/index');
 
 const router = express.Router();
 
+router.get('/:restaurant_id', (req, res) => {
+  const { restaurant_id } = req.params;
+
+  return db.ReservationsTable.findAll(
+    { where: { restaurant_id } },
+  )
+    .then(data => res.status(200).send(data))
+    .catch(err => res.status(500).send(err));
+});
+
+router.post('/:restaurant_id/:reservation_year/:reservation_month/:reservation_date/:reservation_hour/:reservation_min', (req, res) => {
```

Comment:
@LittleJennie 
@Sam-I-Am92 

With this many request params, I might look into going with a query string instead (which will read much more cleanly). You'll see this managed more often via a query string that is updating/managing state as you interact with the page, with a post request being sent directly to that URL when the button is clicked.